docs/how_to/build_mxnet_with_intelmkl.md

```diff
@@ -0,0 +1,114 @@
+# Building MXNet with MKL-based optimizations:
+
+MXNet can be installed and used with several combinations of development 
+tools and libraries on a variety of platforms. This tutorial provides one 
+such recipe describing steps to build and install MXNet with Intel MKL on 
+an Ubuntu-based system, version 16.04.
+
+## Installing
+
+1. Update your operating system and install dependencies:
+
+   ```bash
+    $ sudo apt-get update && apt-get install -y build-essential git libopencv-dev curl gcc libatlas-base-dev python python-pip python-dev python-opencv graphviz python-scipy python-sklearn
+   ```
+
+2. Delete any old or outdated versions of Intel MKL from `/tmp` before 
+   downloading and copying the newest version to `/usr/local`:
+
+   ```bash
+   $ cd /tmp && rm -rf mklml_lnx_*   
+   $ curl -L https://github.com/01org/mkl-dnn/releases/download/v0.10/mklml_lnx_2018.0.20170908.tgz | tar xz
```

Comment:
This seems outdated given that your PR #9218 is already merged.

Reply:
I agree. It should have been changed.